src/main/asciidoc/batch/file-ingest-sftp/local.adoc

```diff
@@ -0,0 +1,209 @@
+║                           │           │     stdout = /var/folders/hd/5yqz2v2d3sxd3n879f4sg4gr0000gn/T/spring-cloud-deployer-120915912946760306/inboundSftp-1540820991695/inboundSftp.task-launcher/stdout_0.log║
+║                           │           │        url = http://192.168.64.1:60081                                                                                                                                 ║
+║                           │           │working.dir = /var/folders/hd/5yqz2v2d3sxd3n879f4sg4gr0000gn/T/spring-cloud-deployer-120915912946760306/inboundSftp-1540820991695/inboundSftp.task-launcher             ║
+╚═══════════════════════════╧═══════════╧════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╝
+
++
+
+. Add data
++
+Normally data would be uploaded to an SFTP server.
+We will simulate this by copying a file into the directory specified by `--remote-dir`.
+Sample data can be found in the `data/` directory of the <<Batch File Ingest>> project.
++
+Copy `data/name-list.csv` into the `/tmp/remote-files` directory which the SFTP source is monitoring.
+When this file is detected, the `sftp` source will download it to the  `/tmp/local-files` directory specified by `--local-dir`, and emit a Task Launch Request.
+The Task Launch Request includes the name of the task to launch along with the local file path, given as the command line argument `localFilePath`.
+Spring Batch binds each command line argument to a corresponding JobParameter.
+The FileIngestTask job processes the file given by the JobParameter named `localFilePath`.
+The `task-launcher` sink polls for messages using an exponential back-off.
+Since there have not been any recent requests, the task will launch within 30 seconds after the request is published.
++
+```
+$ cp data/name-list.csv /tmp/remote-files
+```
+When the batch job launches, you will see something like this in the SCDF console log:
+[source,console,options=nowrap]
+2018-10-26 16:47:24.879  INFO 86034 --- [nio-9393-exec-7] o.s.c.d.spi.local.LocalTaskLauncher      : Command to be executed: /Library/Java/JavaVirtualMachines/jdk1.8.0_60.jdk/Contents/Home/jre/bin/java -jar <path-to>/batch/file-ingest/target/ingest-1.0.0.jar localFilePath=/tmp/local-files/name-list.csv --spring.cloud.task.executionid=1
+2018-10-26 16:47:25.100  INFO 86034 --- [nio-9393-exec-7] o.s.c.d.spi.local.LocalTaskLauncher      : launching task fileIngestTask-8852d94d-9dd8-4760-b0e4-90f75ee028de
+   Logs will be in /var/folders/hd/5yqz2v2d3sxd3n879f4sg4gr0000gn/T/fileIngestTask3100511340216074735/1540586844871/fileIngestTask-8852d94d-9dd8-4760-b0e4-90f75ee028de
++
+
+. Inspect Job Executions
++
+After data is received and the batch job runs, it will be recorded as a Job Execution. We can view job executions by for example issuing the following command in the Spring Cloud Data Flow shell:
```

Comment:
> We can view job executions by for example issuing the following command in the Spring Cloud Data Flow shell:

We can view job executions for example by issuing the following command in the Spring Cloud Data Flow shell: